dashboard/app/views/levels/editors/_playlab.html.haml

```diff
@@ -0,0 +1,82 @@
+- content_for(:head) do
```

Comment:
this file was just renamed from `_studio.html.haml` to `_playlab.html.haml`, right? for whatever reason, github shows this as a new file instead of a rename

Reply:
ya it was just renamed - i dont know why it does that